src/backend.md

```diff
@@ -630,6 +630,55 @@ For serverless projects - method 3 is preferred, but not always. When you have a
       - updateBankAccount.controller.test.ts
 ```
 
+1. One test should include one action as a call of controller or job.
+
+```typescript
+   // bad
+
+   it((caseX) => {
+     // ...
+    await postSqsJob(JobName.name, {...});
+    // ...
+    await postSqsJob(JobName.name, {...});
+   });
+
+   // good
+
+   it((caseX) => {
+     // ...
+    await postSqsJob(JobName.name, {...});
+    // ...
+   });
+
+    it((caseY) => {
+     // ...
+    await postSqsJob(JobName.name, {...});
+    // ...
+   });
+```
+
+1. Test assertions should not verify the results of inner jobs.
```

Comment:
Unclear terminology. What is `inner job`? We can't test jobs? Or do you mean that it should not verify intermediate results? Or do you mean there should be no side effects and we should not test for side effects?